agent/provider/tests/manifest_negotiator.rs

```diff
@@ -133,59 +133,23 @@ fn manifest_negotiator_test_manifest_with_urls(
     None; // error msg
     "Accepted because everyone whitelist is matching even if audited-payload set to none"
 )]
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "none", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "whitelist"}}"#, // rulestore config
     r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule is disabled"); // error msg
+    Some("Everyone rule didn't match whitelist ; Audited-Payload rule whole chain of cert_ids is not trusted"); // error msg
     "Rejected because everyone-whitelist is mismatching and audited-payload set to none"
 )]
-#[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
-    r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    None; // error msg
-    "Accepted because audited-payload all even if everyone-whitelist is mismatching"
-)]
-#[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "whitelist", "description": ""}}}}"#, // rulestore config
-    r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule didn't match whitelist"); // error msg
-    "Rejected because everyone and audited-payload whitelist are mismatching"
-)]
-#[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "none", "audited-payload": {"default": {"mode": "none", "description": ""}}}}"#, // rulestore config
-    r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule is disabled"); // error msg
-    "Rejected because everyone and audited-payload set to none"
-)]
-#[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "none", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
-    r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    None; // error msg
-    "Accepted because audited-payload set to all"
-)]
-#[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "none", "audited-payload": {"default": {"mode": "whitelist", "description": ""}}}}"#, // rulestore config
-    r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule didn't match whitelist"); // error msg
-    "Rejected because audited-payload whitelist doesn't match"
-)]
-#[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "none", "audited-payload": {"default": {"mode": "whitelist", "description": ""}}}}"#, // rulestore config
-    r#"["https://domain.com"]"#, // compManifest.net.inet.out.urls
-    None; // error msg
-    "Accepted because domain is whitelisted when audited-payload set to whitelist"
-)]
```

Comment:
Why these were deleted? Also consider adding more tests for Audited-Payload as logic right now is extended and it is not covered by tests.

Reply:
Right, these could've been easily adapted to the new design, I re-added them.